msteams-platform/messaging-extensions/build-bot-based-plugin.md

```diff
@@ -0,0 +1,242 @@
+---
+title: Create bot based Message extension
+author: v-ypalikila
+description: Learn how to build a bot message extension using Teams Developer Portal and Teams Toolkit.
+ms.localizationpriority: medium
+ms.topic: overview
+ms.author: anclear
+ms.date: 09/07/2023
+---
+
+# Extend bot-based message extension as plugin for Copilot
+
+> [!NOTE]
+> Only bot-based search message extension can be extended as a plugin in Copilot.
+
+Microsoft 365 Copilot, powered by an advanced processing and orchestration engine, integrates Microsoft 365 apps, Microsoft Graph, and large language models (LLMs) to transform your words into a potent productivity tool. Although Copilot can utilize apps and data within the Microsoft 365 ecosystem, many users rely on various external tools and services for work management and collaboration. By extending your message extension as a plugin in Copilot, you can enable users to interact with third-party tools and services, thereby unlocking Microsoft 365 Copilot's full potential. You can achieve this extension by developing a plugin or connecting to an external data source.
+
+## What is a plugin?
+
+A plugin allows Copilot to interact directly with third-party data, apps, and services, enhancing its capabilities and broadening its range of capabilities. Plugins allow Copilot to:
+
+* Fetch real-time information, such as the latest news coverage on a product launch.
+* Retrieve knowledge-based information, such as a team’s design files in Figma.
+* Perform actions on behalf of the user, such as creating a Contoso ticket.
+
+All Bot-based search message extensions are eligible for plugins support, subject to validation to ensure the plugin meets quality, security, privacy, and usefulness expectations.
+
+You can create a bot-based message extension using Teams Toolkit for Visual Studio Code, Visual Studio, or CLI and extend the extension to function as a plugin in Copilot.
+
+## Prerequisites
+
+Before you get started, ensure that you meet the following pre-requisites:
```

Comment:
Since these are introduced as prereqs to getting started building an ME plugin, how about saying "ensure that you're familiar with the following standards and guidelines for building message extension Copilot plugins". (I don't think you can actually "meet" these unless you already have implemented an ME plugin?)

Reply:
Updated.